README.md

```diff
@@ -191,7 +191,7 @@
 |------|:----------:|:--------:|
 | [Combo Meals](https://www.hackerrank.com/challenges/combo-meal) | Easy | |
 | [Difference and Product](https://www.hackerrank.com/challenges/difference-and-product) | Easy | |
-| [Pythagorean Triplet](https://www.hackerrank.com/challenges/pythagorean-triple) | Easy | |
+| [Pythagorean Triplet](https://www.hackerrank.com/challenges/pythagorean-triple) | Easy|[![cpp](https://img.icons8.com/color/35/000000/c-plus-plus-logo.png)](cpp/pythagoreanTriplet.cpp) | |
```

Comment:
Thanks for solving this 💯 when you add the image logo link make sure to add it in the `| |` which is already there as currently there is an extra `| |` after the link. 

also when you solve a question in any language - this case cpp, update the badge in the `README.md` file as well. 

so `![problems-solved-cpp](https://img.shields.io/badge/C++-1/284-008000.svg)` --> `![problems-solved-cpp](https://img.shields.io/badge/C++-2/284-008000.svg)`

Also update the Problems solved badge if the you are solving a new problem that wasn't solved before. 
`![problems-solved](https://img.shields.io/badge/Problems%20Solved-2/284-1f425f.svg)` --> `![problems-solved](https://img.shields.io/badge/Problems%20Solved-3/284-1f425f.svg)`

Thanks again 😃